plugins/fontpreview.lua

```diff
@@ -0,0 +1,63 @@
+--mod-version:3 --priority:110
+
+local common = require 'core.common'
+local style = require 'core.style'
+local View = require 'core.view'
+
+local FontView = View:extend()
+
+local font_text = "The quick brown fox jumps over the lazy dog."
+
+function FontView:new(path)
+	FontView.super.new(self)
+	self.path = path
+	self.fonts = {}
+	for i=1, 8 do self.fonts[i] = renderer.font.load(path, 12+i*7) end
+	self.scrollable = true
+end
+
+function FontView:get_h_scrollable_size()
+	return self.fonts[#self.fonts]:get_width(font_text) + style.padding.x
+end
+
+function FontView:get_scrollable_size()
+	return math.huge
+end
+
+function FontView:get_name()
+	return "Font: " .. self.path
```

Comment:
```suggestion
	return "Font: " .. common.basename(self.path)
```
might be a good idea.

Reply:
you might want to open a font that's outside the project folder (for whatever reason), so it shows the full path, and a relative path if the font is otherwise in the project